tasks/pipeline-osdeps-get-latest-mediainfo-url.yml

```
@@ -1,28 +1,25 @@
-- name: "Get the latest release of MediaInfo from GitHub"
-  uri:
-    url: https://api.github.com/repos/MediaArea/MediaInfo/releases/latest
-    method: GET
-    headers:
-      Accept: application/vnd.github.v3+json
-  register: __mediainfo_result
-
+- name: "mediaarea repo: Fetch the repos.html.twig file from GitHub"
+  ansible.builtin.uri:
+    url: https://raw.githubusercontent.com/MediaArea/MediaArea-Website/master/src/AppBundle/Resources/views/Default/repos.html.twig
+    return_content: yes
+  register: repo_page
 
-- name: "Extract MediaInfo major version number"
-  set_fact:
-    __mediainfo_major_version: "{{ (__mediainfo_result.json.tag_name | regex_search('^[vV]?(\\d+)', '\\1') | first) | string }}"
+- name: "mediaarea repo: Extract the latest deb package URL"
+  ansible.builtin.set_fact:
+    __mediainfo_latest_deb_url: "{{ repo_page.content | regex_search('https:\\/\\/mediaarea\\.net\\/repo\\/deb\\/repo-mediaarea_\\S+\\.deb') }}"
 
-- name: "Check if the repo package URL is valid"
+- name: "mediaarea repo: Check if the repo package URL is valid"
   uri:
-    url: "https://mediaarea.net/repo/deb/repo-mediaarea_1.0-{{ __mediainfo_major_version }}_all.deb"
+    url: "{{ __mediainfo_latest_deb_url }}"
     method: HEAD
   register: __mediainfo_url_response
 
-- name: "Set variable if URL when package exists"
+- name: "mediaarea repo: Set variable if URL when package exists"
```

This reads a bit strange. How about:

```suggestion
- name: "mediaarea repo: Set variable if URL is valid"
```

---

Sorry! I'll add the change in next commit. Thanks!